### DISCUSSION TITLE
where should I have the sasl jaas.config

### DISCUSSION THREAD

**Discussion by Ge0f3** [provectus/kafka-ui#2715] [Q&A] [answered]
My application.yaml looks like below      
    ssl.truststore.location: 
      ssl.truststore.password:
      sasl:
        mechanism: 
        jaas.config: 

and my docker run command 

docker run --rm  -p 8080:8080 \
    -e KAFKA_CLUSTERS_0_PROPERTIES_SECURITY_PROTOCOL= SASL_SSL \
    -e KAFKA_CLUSTERS_0_BOOTSTRAPSERVERS=asfasdf \
    -e KAFKA_CLUSTERS_0_PROPERTIES_SSL_TRUSTSTORE_LOCATION= \
    -e KAFKA_CLUSTERS_0_PROPERTIES_SSL_TRUSTSTORE_PASSWORD=\
     provectuslabs/kafka-ui:latest 

where should I provide the sasl mechanish and jaas config

**Comment by Haarolean** [comment, marked as answer]
Hey, thanks for reaching out.

Please take a look at [this example](https://github.com/provectus/kafka-ui/blob/master/documentation/compose/kafka-ui-sasl.yaml#L20) with jaas properties.
